modules/xmpp/ChatRoom.js

```diff
@@ -892,9 +892,10 @@ export default class ChatRoom extends Listenable {
      * Send text message to the other participants in the conference
      * @param message
      * @param elementName
+     * @param {string} [roomjid] the jid of the chat room where the message will be sent
      */
-    sendMessage(message, elementName) {
-        const msg = $msg({ to: this.roomjid,
+    sendMessage(message, elementName, roomjid = this.roomjid) {
```

Comment:
I don't think this is the right thing to do.

Reply:
reverted as requested